backend/src/handlers.rs

```diff
@@ -0,0 +1,41 @@
+use crate::gamestate::game_repo::{GameRepo, StableGameRepo};
+use actix_web::{post, web, HttpResponse, Responder};
+use serde::Deserialize;
+use serde::Serialize;
+use std::sync::{Arc, Mutex};
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct GamePostData {
+    name: String,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Response {
+    gameID: String,
+    server: String,
+    token: String,
+}
+
+#[post("/game")]
+pub async fn create_game(
+    data: web::Data<Arc<Mutex<StableGameRepo>>>,
+    body: web::Json<GamePostData>,
+) -> impl Responder {
+    let result = data.lock().unwrap().create_game(body.name.clone()).await;
+    match result {
+        Ok(_) => HttpResponse::Created().json(Response {
+            gameID: result.as_ref().unwrap().id.clone(),
+            server: "127.0.0.1:9000".to_string(),
```

Comment:
Adresu serveru prosím buď na nějaké jednotné místo, odkud bude možné ji snadno upravit, nebo (lepší varianta) jako CLI argument předat serveru číslo portu a zbytek si zjistit od systému.

Reply:
Also možná nice-to-have do budoucna by bylo zkoušet více portů, než najdeme nějaký volný. Byl by celkem fail, kdyby nám při obhajobě projektu nešel spustit server, protože si port 9000 zablokovala nějaká random appka :D

Reply:
Vyřešeno, sken vícero portů zařazen mezi issues :)